docs/DeveloperGuide.adoc

```diff
@@ -257,6 +257,59 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g App name, logging level) through the configuration file (default: `config.json`).
 
+== implementation
+
+=== timetable feature
+==== current implementation
+
+The `timetable` is contained in `person` which is in `model`.
+
+* `add_timetable` -- adds a timetable from the csv file to the person.
+* `delete_timetable` -- delete a timetable from the person and creates a default timetable in person. It will also find the timetable csv file of the person and deletes it.
+* `download_timetable` -- downloads a timetable into a csv file into a stored folder location from the person.
+
+These operations are handled by the `logic` component and uses the `person` in `model` to do the execution.
+
+Given below is an example usage scenario and how the `add_timetable` behaves at each step.
+
+Step 1: The user launches the application for the first time. The NUS Hangs will initialise the person with the timetable with the details which was stored using Storage.
+
+image:: addTimetable-step-1.png[width="300"]
+
+Step 2:  user opens a timetable of first `person` in the stored location and edits it via microsoft excel or with his preferred software.
+
+image:: addTimetable-step-2.png[width="300"]
```

Comment:
The image is not displaying. I think you need to add caption e.g.

.Caption
image::image.png[width="800"]

Reply:
ok thanks